package.json

```diff
@@ -57,8 +57,8 @@
     "@angular/router": "^18.0.1",
     "@angular/service-worker": "^18.0.1",
     "@angular/ssr": "^18.0.2",
-    "@commitlint/cli": "^19.0.0",
-    "@commitlint/config-angular": "^19.0.0",
+    "@commitlint/cli": "^13.0.0",
+    "@commitlint/config-angular": "^13.0.0",
```

Comment:
Due to dependency `conventional-changelog-angular` is v7, which is conflict with dependency of `conventional-changelog-cli`, I failed to generate changelog.
Here I revert the version to preview v13, and I'll solve it later